src/test/java/com/game/Battleship/controllers/BoardControllerTests.java

```diff
@@ -18,27 +23,31 @@
     private TestRestTemplate template;
 
     @Test
-    public void boardEndpointContainsSizeAttribute(){
+    public void boardsEndpointContains2Boards() {
```

Comment:
Is it convention to use an integer in a method name?

Reply:
changed !